inference/server/oasst_inference_server/routes/workers.py

```diff
@@ -340,9 +340,12 @@ async def handle_generated_text_response(
         message_id = work_response_container.message_id
         async with deps.manual_create_session() as session:
             cr = chat_repository.ChatRepository(session=session)
+            work_parameters = work_response_container.work_request.parameters
+            work_parameters = work_parameters.copy(update={"used_plugin": response.used_plugin})
```

Comment:
does "used" here mean that the model actually chose to use it? if yes, could we store this somewhere else than in the work parameters? because this is more a response-value than a request-value

Reply:
yes the `work_parameters` are meant as input, i.e. if I send them somewhere else I should get the same output, therefore we don't want response values in here. I see that this makes it easy to show in the frontend, but we'll have to find another way of doing that. I think we could store it directly in the message object and default to none, similar to the safety properties that have been added recently